### PR TITLE
Add docs for HTTP API Gateway

### PR DESCRIPTION
**Pull request by deliahu**

### CODE REVIEW

File: docs/clusters/networking/api-gateway.md [hunk 1]
```diff
@@ -12,7 +12,69 @@ If your API load balancer is internal (i.e. you set `api_load_balancer_scheme: i
...
+
+Click "Add integration".
+
+![](https://user-images.githubusercontent.com/808475/125668635-f92df672-f516-45e0-a152-538aff901c0d.png)
+
+Click the drop-down menu, and select "HTTP".
+
+![](https://user-images.githubusercontent.com/808475/125668655-78754a51-c77a-4a03-a548-78ab991d1486.png)
+
+Set "Integration URL" to "http://API_LOAD_BALANCER_ENDPOINT/{proxy}". You can get your API load balancer endpoint via `cortex cluster info`; make sure to prepend `http://` and append `/{proxy}`. For example, mine is: `http://aa9f5fdabfa6446aca53a526f59bc3c5-18cd00a628421fa3.elb.us-east-1.amazonaws.com/{proxy}`.
```
Comment (RobertLucian) [inline docs/clusters/networking/api-gateway.md:33]: What do you say about wrapping "http://API_LOAD_BALANCER_ENDPOINT/{proxy}" like this `"http://API_LOAD_BALANCER_ENDPOINT/{proxy}"`? Same everywhere else.

Comment (RobertLucian) [inline docs/clusters/networking/api-gateway.md:33]: In the screenshot following this, I see the "Integration URL" as "URL endpoint". Is that okay? Or is there a pop-up in between the screenshots?

File: docs/clusters/networking/api-gateway.md [hunk 2]
```diff
@@ -12,7 +12,69 @@ If your API load balancer is internal (i.e. you set `api_load_balancer_scheme: i
 
 _This section applies if your API load balancer is internet-facing (which is the default, or you set `api_load_balancer_scheme: internet-facing` in your cluster configuration file before creating your cluster). If your API load balancer is internal, see the [internal load balancer](#internal-load-balancer) section below._
 
-### Create an API Gateway
+There are two types of API Gateways that can be used when your load balancer is internet-facing: HTTP and REST. HTTP APIs are faster and less expensive, while REST APIs have more features an allow for more control. See the following section for creating an [HTTP API](#http-api), or skip to the next section for creating a [REST API](#rest-api).
```
Comment (RobertLucian) [inline docs/clusters/networking/api-gateway.md:15]: Maybe we should add a note about the necessity of having an HTTP load balancer for the Grafana endpoint. Maybe we can issue a recommendation of only using the HTTP load balancer on the operator ingress. What do you think?

Comment (deliahu) [reply docs/clusters/networking/api-gateway.md:15]: As discussed, we will hold off for now

**Review by RobertLucian — APPROVED** [review]

**Pull request merged**